Client/mods/deathmatch/logic/CClientPed.h

```diff
@@ -553,6 +553,8 @@ class CClientPed : public CClientStreamElement, public CAntiCheatModule
 
     std::unique_ptr<CAnimBlendAssociation> GetAnimAssociation(CAnimBlendHierarchySAInterface* pHierarchyInterface);
 
+    void SetHasSyncedAnim(bool synced) noexcept { m_hasSyncedAnim = synced; };
+    bool HasSyncedAnim() const noexcept { return m_hasSyncedAnim; };
```

Comment:
Removed the semicolon after the last brace.
```suggestion
    void SetHasSyncedAnim(bool synced) noexcept { m_hasSyncedAnim = synced; }
    bool HasSyncedAnim() const noexcept { return m_hasSyncedAnim; }
```